src/read.c

```diff
@@ -5775,7 +5778,7 @@ static avifResult avifDecoderFindGainMapItem(const avifDecoder * decoder,
     avifTransferCharacteristics transferCharacteristics = AVIF_TRANSFER_CHARACTERISTICS_UNSPECIFIED;
     avifMatrixCoefficients matrixCoefficients = AVIF_MATRIX_COEFFICIENTS_UNSPECIFIED;
     avifRange yuvRange = AVIF_RANGE_FULL;
-    avifBool cicpSet;
+    avifBool cicpSet = AVIF_FALSE;
```

Comment:
This change is temporary. We need to merge either https://github.com/AOMediaCodec/libavif/pull/2897 or https://github.com/AOMediaCodec/libavif/pull/2898 as a prerequisite for this PR.

Reply:
https://github.com/AOMediaCodec/libavif/pull/2898 has been merged.